tests/AutoReview/DescribeCommandTest.php

```diff
@@ -1,64 +0,0 @@
-<?php
-
-/*
- * This file is part of PHP CS Fixer.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
- *
- * This source file is subject to the MIT license that is bundled
- * with this source code in the file LICENSE.
- */
-
-namespace PhpCsFixer\Tests\AutoReview;
-
-use PhpCsFixer\Console\Application;
-use PhpCsFixer\Console\Command\DescribeCommand;
-use PhpCsFixer\FixerFactory;
-use PHPUnit\Framework\TestCase;
-use Symfony\Component\Console\Tester\CommandTester;
-
-/**
- * @internal
- *
- * @coversNothing
- * @group auto-review
- */
-final class DescribeCommandTest extends TestCase
```

Comment:
- `PhpCsFixer\Tests\Console\Command\DescribeCommandTest`: generic tests with fake data;
- `PhpCsFixer\Tests\AutoReview\DescribeCommandTest`: tests that check the command runs fine with all real fixers.

The separation was made on purpose but I don't remember where we discussed that. Though I agree this is a but confusing so I'm fine with a single class.